servers/rendering/renderer_rd/shaders/effects/blur_raster.glsl

```diff
@@ -110,7 +123,10 @@ void main() {
 		GLOW_ADD(vec2(0.0, 2.0), 0.122581);
 		GLOW_ADD(vec2(0.0, -1.0), 0.233062);
 		GLOW_ADD(vec2(0.0, -2.0), 0.122581);
-		color *= blur.glow_strength;
+
+		// this would double this up...
+		// color *= blur.glow_strength;
```

Comment:
``glow_strength`` should get applied below in the ``FLAG_GLOW_FIRST_PASS`` block as it is in the Compute version

Reply:
No, it doesn't, remember the compute version doesn't split each pass into two passes, the fragment shader needs to do a horizontal pass first, then a vertical, so if we move it down we and up applying it twice making everything twice as bright.